tests/storage/test_receipts.py

```diff
@@ -290,3 +290,76 @@ def test_get_last_receipt_event_id_for_user(self) -> None:
             )
         )
         self.assertEqual(res, event2_1_id)
+
+    def test_receipts_clear_push_actions(self) -> None:
```

Comment:
Could you add a docstring explaining what this test tests please? Same with the other test.